src/elements/GitHubCorner/GitHubCorner.md

```diff
@@ -45,7 +45,7 @@ A GitHub corner without color is black.
 ```
 
 GitHub corners can be colored with any color from the set provided by
-CC Vocabulary. Black, in one of rare cases like this, looks best though.
+Vue Vocabulary. Black, in one of rare cases like this, looks best though.
```

Comment:
In places like this, we'd rather have just 'Vocabulary'.

Reply:
Got it. I got confused between the design system and specific implementation.